config/environments/test.rb

```diff
@@ -33,6 +35,15 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Raises error for missing translations.
-  config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
I think this needs to be uncommented, as we previously had `raise_on_missing_translations` switched on.